scripts/buildkite/default.nix

```diff
@@ -1,22 +1,40 @@
 { system ? builtins.currentSystem
+, crossSystem ? null
```

Comment:
The `scrips/buildkite` directory could be deleted, right?

Reply:
I left the `cabal check` buildkite script there, but can delete if preferred.

Reply:
cabal check is removed.

Reply:
Build failed when I removed it so I put it back.  Maybe remove it in a future PR.